src/language-css/print/comma-separated-value-group.js

```diff
@@ -269,7 +269,8 @@ function printCommaSeparatedValueGroup(path, options, print) {
       options.parser === "scss" &&
       isMathOperator &&
       iNode.value === "-" &&
-      iNextNode.type === "value-func"
+      iNextNode.type === "value-func" &&
+      (options.parser !== "scss" || locEnd(iNode) !== locStart(iNextNode))
```

Comment:
Didn't notice the parser already checked above.

Reply:
Ohh.. I missed that too